docs/usage/getting-started.rst

```diff
@@ -0,0 +1,95 @@
+Getting Started
+===============
+
+Install Clowder
+---------------
+
+As covered in the main README section [1]_ , the installation process for Clowder is a breeze. 
```

Comment:
We should probably mention which type of environment we are talking about - installing it locally......